Description: Bring ddupdate.service into shape.
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 ddupdate (0.0.5~rc2-2) experimental; urgency=medium
 .
   *  Added upstream  fix for NetworkManager support.
Author: Alec Leamas <leamas.alec@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2018-01-04

--- ddupdate-0.0.5~rc2.orig/systemd/ddupdate.service
+++ ddupdate-0.0.5~rc2/systemd/ddupdate.service
@@ -6,7 +6,7 @@ After=network.target
 # User=ddupdate
 Type=oneshot
 Environment=PATH=/bin:/usr/bin:/sbin:/usr/sbin
-ExecStart=/usr/bin/ddupdate
+ExecStart=/usr/local/bin/ddupdate
 
 [Install]
 WantedBy=multi-user.target
